tools/juliapkg/src/old_interface.jl

```diff
@@ -19,3 +19,14 @@ end
 
 appendDataFrame(input_df::DataFrame, db::DB, table::AbstractString, schema::String = "main") =
     appendDataFrame(input_df, db.main_connection, table, schema)
+
+"""
+    DuckDB.load!(input_df::DataFrame, con, table)
+
+Load an input DataFrame `input_df` into a DuckDB table that will be named `table`.
+"""
+function load!(con::Connection, input_df::DataFrame, table::AbstractString, schema::String = "main")
+    register_data_frame(con, input_df, "__append_df")
+    DBInterface.execute(con, "CREATE TABLE \"$schema\".\"$table\" AS SELECT * FROM __append_df")
+    return unregister_data_frame(con, "__append_df")
```

Comment:
Is this supposed to be the return value?

Reply:
That's what `append_data_frame` had done before. As `unregister_data_frame` just returns nothing, I made that more explicit now.